tests/_utils/view.js

```diff
@@ -409,26 +470,37 @@ class ViewParser {
 	 * Parses HTML-like string to view tree elements.
 	 *
 	 * @param {String} data
+	 * @param {engine.treeView.Element|engine.treeView.DocumentFragment} [rootElement=null] Default root to use when parsing elements.
+	 * When set to `null` root element will be created automatically. If set to
+	 * {@link engine.treeView.Element Element} or {@link engine.treeView.DocumentFragment DocumentFragment} - this node
+	 * will be used as root for all parsed nodes.
 	 * @returns {engine.treeView.Node|engine.treeView.DocumentFragment}
 	 */
-	parse( data ) {
+	parse( data, rootElement = null ) {
 		const htmlProcessor = new HtmlDataProcessor();
 
 		// Convert HTML string to DOM.
 		const domRoot = htmlProcessor.toDom( data );
 
+		// If root element is provided - clear it.
+		// if ( rootElement ) {
+		// 	rootElement.removeChildren( rootElement.getChildCount() );
```

Comment:
It shouldn't :) In model, removing items in root element is happening in `parse` method, so the same should happen here in the view. Now it is happening in `setData`. I totally forgotten about this. Thanks.